single-node/run-performance-tests.sh

```diff
@@ -120,105 +120,105 @@ declare -A test_scenario9=(
-    [jmx]="oauth/OAuth_AuthCode_Redirect_WithConsent.jmx"
-    [tenantMode]=true
-    [skip]=true
-    [modes]="FULL"
-)
-declare -A test_scenario11=(
-    [name]="11-oauth_implicit_redirect_with_consent_tenant"
-    [display_name]="Implicit Grant Redirect With Consent"
-    [description]="Obtain an access token using the OAuth 2.0 implicit grant type."
-    [jmx]="oauth/OAuth_Implicit_Redirect_WithConsent.jmx"
-    [tenantMode]=true
-    [skip]=true
-    [modes]="FULL"
-)
-declare -A test_scenario12=(
-    [name]="12-oauth_password_grant_tenant"
-    [display_name]="Password Grant Type"
-    [description]="Obtain an access token using the OAuth 2.0 password grant type."
-    [jmx]="oauth/OAuth_Password_Grant.jmx"
-    [tenantMode]=true
-    [skip]=true
-    [modes]="FULL"
-)
-declare -A test_scenario13=(
-    [name]="04-oauth_client_credential_grant_tenant"
-    [display_name]="Client Credentials Grant Type"
-    [description]="Obtain an access token using the OAuth 2.0 client credential grant type."
-    [jmx]="oauth/OAuth_Client_Credentials_Grant.jmx"
-    [tenantMode]=true
-    [skip]=true
-    [modes]="FULL"
-)
-declare -A test_scenario14=(
-    [name]="14-oidc_auth_code_redirect_with_consent_tenant"
-    [display_name]="OIDC Auth Code Grant Redirect With Consent"
-    [description]="Obtain an access token and an id token using the OAuth 2.0 authorization code grant type."
-    [jmx]="oidc/OIDC_AuthCode_Redirect_WithConsent.jmx"
-    [tenantMode]=true
-    [skip]=true
-    [modes]="FULL"
-)
-declare -A test_scenario15=(
-    [name]="15-oidc_implicit_redirect_with_consent_tenant"
-    [display_name]="OIDC Implicit Grant Redirect With Consent"
-    [description]="Obtain an access token and an id token using the OAuth 2.0 implicit grant type."
-    [jmx]="oidc/OIDC_Implicit_Redirect_WithConsent.jmx"
-    [tenantMode]=true
-    [skip]=true
-    [modes]="FULL"
-)
-declare -A test_scenario16=(
-    [name]="16-oidc_password_grant_tenant"
-    [display_name]="OIDC Password Grant Type"
-    [description]="Obtain an access token and an id token using the OAuth 2.0 password grant type."
-    [jmx]="oidc/OIDC_Password_Grant.jmx"
-    [tenantMode]=true
-    [skip]=true
-    [modes]="FULL"
-)
-declare -A test_scenario17=(
-    [name]="17-oidc_request_path_authenticator_tenant"
-    [display_name]="OIDC Auth Code Request Path Authenticator With Consent"
-    [description]="Obtain an access token and an id token using the request path authenticator."
-    [jmx]="oidc/OIDC_AuthCode_Request_Path_Authenticator_WithConsent.jmx"
-    [tenantMode]=true
-    [skip]=true
-    [modes]="FULL"
-)
-declare -A test_scenario18=(
-    [name]="18-saml2_sso_redirect_binding_tenant"
-    [display_name]="SAML2 SSO Redirect Binding"
-    [description]="Obtain a SAML 2 assertion response using redirect binding."
-    [jmx]="saml/SAML2_SSO_Redirect_Binding.jmx"
-    [tenantMode]=true
-    [skip]=true
-    [modes]="FULL"
-)
+#declare -A test_scenario10=(
```

Comment:
why these are commented

Reply:
The commented tests are run on tenanted mode, since we are running the performance test on the super tenant, these test scenarios are commented.